com.unity.probuilder/Editor/EditorCore/EditorSceneViewPicker.cs

```diff
@@ -106,7 +107,21 @@ public static ProBuilderMesh DoMouseClick(Event evt, SelectMode selectionMode, S
                     UndoUtility.RecordSelection(mesh, "Select Vertex");
 
                     if (ind > -1)
-                        mesh.SetSelectedVertices(mesh.selectedIndexesInternal.RemoveAt(ind));
+                    {
+                        var sharedIndex = mesh.sharedVertexLookup[s_Selection.vertex];
+                        var sharedVertex = mesh.sharedVerticesInternal[sharedIndex];
+                        s_IndexBuffer.Clear();
+                        foreach (var vertex in sharedVertex)
+                        {
+                            var index = Array.IndexOf(mesh.selectedIndexesInternal, vertex);
+                            if (index < 0)
+                                continue;
+
+                            s_IndexBuffer.Add(index);
+                        }
```

Comment:
This could be condensed to `mesh.GetCoincidentVertices(s_Selection.vertex, s_IndexBuffer);`

Reply:
I need the index of the vertex in the selected indexes array and not the index of the vertex itself.